pkg/sql/catalog/externalcatalog/external_catalog_test.go

```diff
@@ -48,7 +49,12 @@ func TestExtractIngestExternalCatalog(t *testing.T) {
 	sysDB := sqlutils.MakeSQLRunner(srv.SystemLayer().SQLConn(t))
 	sysDB.Exec(t, "SET CLUSTER SETTING sql.virtual_cluster.feature_access.zone_configs.enabled = true;")
 	sysDB.Exec(t, "SET CLUSTER SETTING sql.virtual_cluster.feature_access.zone_configs_unrestricted.enabled = true;")
-	sysDB.Exec(t, "SET CLUSTER SETTING sql.gc_job.wait_for_gc.interval = '250ms'")
+
+	rng, _ := randutil.NewTestRand()
+	fastGC := rng.Float64() < 0.5
+	if fastGC {
```

Comment:
What's the motivation for making this probabilistic? The two different modes and assertions increase the risk of the test flaking.

Reply:
added a bit more context to the commit message. thoughts?